indexer/services/comlink/__tests__/controllers/api/v4/compliance-v2-controller.test.ts

```diff
@@ -633,5 +634,30 @@ describe('ComplianceV2Controller', () => {
       expect(response.body.reason).toEqual(ComplianceReason.US_GEO);
       expect(response.body.updatedAt).toBeDefined();
     });
+
+    it('Stats being sent for the geoblock compliance', async () => {
```

Comment:
nit: just add this expectation on all of the other tests, instead of having a separate test for this. If you're looking for something especially clean, follow [this example](https://github.com/dydxprotocol/v4-chain/blob/9d56ab8d3dc8e37a69612713048ddb9bdc5bd2d4/indexer/services/ender/__tests__/handlers/order-fills/liquidation-handler.test.ts#L1204)